.github/workflows/sonar-cloud-static-analysis.yml

```diff
@@ -23,6 +23,7 @@ jobs:
       - name: Set up JDK 11
         uses: actions/setup-java@v3
         with:
+          distribution: 'temurin'
```

Comment:
Two questions:

1) Is this configuration default in play for the version of sonar we're using? Should we bump our version (4.7.0.2747) above to take advantage of this?

2) The commit message doesn't mention changing the Java distribution. What's the benefit of using "temurin" vs the default from the action? If there's some measurable improvements we get to the action from that change it should be mentioned in the commit (or not made at all if it's just arbitrary).

Reply:
@ADKaster `distribution` is mandatory, the SonarCloud analysis has been failing for 2 months: https://github.com/SerenityOS/serenity/actions/runs/4537852241/jobs/7996140859

Reply:
Whoops :smiling_face_with_tear: